### DISCUSSION TITLE
Way to make `dq` to work outside of fugitive buffer?

### DISCUSSION THREAD

**Discussion by xiaochuanyu** [tpope/vim-fugitive#2046] [Q&A] [answered]
Is there some way of mapping `dq` which cleans up diff buffers for me to work outside of the `fugitive` buffer?
After reading I tried below after reading https://github.com/tpope/vim-fugitive/issues/1425 without success:
```
nmap <leader>gg <Plug>fugitive:dq
```
In my workflow I run `:Git difftool -y` to look at diffs for all my files. Then, when I'm done looking I'd like to just do `<leader>gg` anywhere to clean up all the diff buffers (and ideally those extra tabs too).

**Comment by tpope** [comment, marked as answer]
The underlying function is public so you can call it directly:

```
nmap <silent><script> <Leader>gg :<C-U>call fugitive#DiffClose()<CR>
```

This won't touch other tabs; Fugitive doesn't have anything for that.
